### PR TITLE
Connect imdb api

### PR DESCRIPTION
**Pull request by JaturongKongmanee**

### CODE REVIEW

File: modules/movies.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        Genre: ${info.Genre},
+        Director: ${info.Director}`
+
+    return template;
+}
+
+
+async function fetchMoviesInfo(name) {
+    const API_KEY = `63c12de2`;
+    let url=`http://www.omdbapi.com/?t=${name}&apikey=${API_KEY}`
```
Comment (smiile8888) [inline modules/movies.js:27]: Move `API_KEY` to `.env` then call it using `process.env.MOVIE_API_KEY`

**Note:** Don't forget to import `dotenv` and do `dotenv.config()`, please refer to `index.js` or `app.js`

Comment (smiile8888) [reply modules/movies.js:27]: `url` can be `const`

**Comment by JaturongKongmanee** [comment]
@smiile8888 where is it?

**Comment by smiile8888** [comment]
> @smiile8888 where is it?

in `index.js`

**Comment by JaturongKongmanee** [comment]
@smiile8888  just leave the comment there!

File: index.js [hunk 2]
```diff
@@ -1,16 +1,52 @@
...
+    if (cmd === cli.GG) {
+        console.log(msg)
+        msg.reply("GG Pong!");
+    }
+
+    if (cmd === cli.MOVIE) {
+        movie.fetchMoviesInfo(argument).then((res) => {
+            msg.reply(res);
+        });
+    }
```
Comment (smiile8888) [inline index.js:49]: @JaturongKongmanee This part

File: index.js [hunk 3]
```diff
@@ -1,16 +1,48 @@
...
 const dotenv = require('dotenv');
+const movie = require('./modules/movie-fetch.js');
+
 dotenv.config();
 
+const cli = Object.freeze({
+    MOVIE: `/movie`,
+    PING: `ping`,
+    GG: `gg`,
+});
```
Comment (smiile8888) [inline index.js:12]: I like this! 🎉

**Pull request merged**